src/views/landing.vue

```diff
@@ -73,9 +73,11 @@ export default class Landing extends Vue {
 
   async mounted () {
     const { id } = this
-    // if an id was specified then get and load the subject NR
-    if (id) {
-      const nrData = await NamexServices.getNameRequest(true)
+    const accountId = this.$route?.query?.accountid?.toString()
```

Comment:
When is `$route` ever not an object?

Reply:
Don't know.... I just copied it from other spot.

Reply:
I believe it should never be null (even in the other spot), except if a unit test runs without setting it first.

Conditional chaining is great except it can hide some errors. For example, you might have a fallback action if the chain is undefined, but maybe you should be doing a different action if the first part of the chain is valid. That's why I don't like to have question marks on all parts of a chain.